components/form/Select.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Label, ErrorMessage } from '.';
+import { useFormContext } from 'react-hook-form';
+
+export function SelectField({ className, containerClassName, label, labelClassName, name, ...options }) {
```

Comment:
The `...options` that you used here is not explanatory enough, can we use the destructure the  exact props we need and destructure the rest as `...props`